### PR TITLE
Add trimmed down OC Agent to receive only OpenCensus proto spans

### PR DESCRIPTION
**Pull request by ChrisCoe**
I copied over the OpenCensus agent and removed receivers and exporters that were not related to the OpenCensus Receiver or my exporter. I was able to reduce the agent from over 150 MB to now 20 MB. This is still quite large, and can use more trimming, but for my demo this should suffice for now.

The next PR will be modifying this agent to send OC proto spans to 1Agent. 

@reyang

### CODE REVIEW

File: OpenCensusAgentPluggin/opencensus-service/cmd/ocagent/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Use a helper to create an emtpy configuration since the agent requires such file
```
Comment (simathih) [inline OpenCensusAgentPluggin/opencensus-service/cmd/ocagent/Dockerfile]: *empty

**Review by simathih — COMMENTED** [review]
Please rename OpenCensusAgentPluggin to just OpenCensusAgent

**Review by simathih — CHANGES_REQUESTED** [review]
How many files are there? Do we need all of them?

File: OpenCensusAgentPluggin/opencensus-service/exporter/loggingexporter/logging_exporter_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
+// Copyright 2019, OpenCensus Authors
```
Comment (simathih) [inline OpenCensusAgentPluggin/opencensus-service/exporter/loggingexporter/logging_exporter_test.go]: it doesn't seem like we need these tests, can you exclude these too?

File: OpenCensusAgentPluggin/opencensus-service/internal/configv2/testdata/missing-processor-type.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+receivers:
```
Comment (simathih) [inline OpenCensusAgentPluggin/opencensus-service/internal/configv2/testdata/missing-processor-type.yaml]: pls exclude the testdata folder

File: OpenCensusAgentPluggin/opencensus-service/exporter/loggingexporter/logging_exporter_test.go (hunk 2 above)
Comment (ChrisCoe) [reply OpenCensusAgentPluggin/opencensus-service/exporter/loggingexporter/logging_exporter_test.go]: Yeah, I wasn't sure if I should take out all the tests. That is something I can definitely do.

File: OpenCensusAgentPluggin/opencensus-service/internal/factories/factories_test.go [hunk 4]
```diff
@@ -0,0 +1,200 @@
+// Copyright 2019, OpenCensus Authors
+//
```
Comment (simathih) [inline OpenCensusAgentPluggin/opencensus-service/internal/factories/factories_test.go]: this seems like a test too

**Comment by ChrisCoe** [comment]
I took it down from 150 to 51 files. We don't need all these files. From my understanding of the agent, there are probably only 20 files that are esscential for the OC receiver to function. I can reduce it more, but it's kinda tricky for me at this point. Should I keep trimming?

**Review by simathih — APPROVED** [review]

**Pull request merged**